packages/python-packages/apiview-copilot/src/_apiview_reviewer.py

```diff
@@ -18,12 +23,15 @@
 _GUIDELINES_FOLDER = os.path.join(_PACKAGE_ROOT, "guidelines")
 _PROMPTS_FOLDER = os.path.join(_PACKAGE_ROOT, "prompts")
 
-DEFAULT_MODEL = "o3-mini"
+COSMOS_ACC_NAME = os.environ.get("AZURE_COSMOS_ACC_NAME")
+COSMOS_DB_NAME = os.environ.get("AZURE_COSMOS_DB_NAME")
+COSMOS_ENDPOINT = f"https://{COSMOS_ACC_NAME}.documents.azure.com:443/"
+CREDENTIAL = DefaultAzureCredential()
```

Comment:
Doesn't have to be in this PR, but we should probably document these env vars in the project readme.

Reply:
Updated the README. It was very, VERY out-of-date!